packages/installer/src/gcp.ts

```diff
@@ -58,18 +53,12 @@ export function* ensureGCPInfraExists(
     throw Error("`gcp` property expected");
   }
 
-  if (ccfg.custom_dns_name === undefined) {
-    const dnsConf = getDnsConfig(ccfg.cloud_provider);
-    const dnsname = yield call(ensureDNSExists, {
-      opstraceClusterName: ccfg.cluster_name,
-      dnsName: dnsConf.dnsName,
-      target: ccfg.cloud_provider,
-      dnsProvider: ccfg.cloud_provider
-    });
-
```

Comment:
Note: might want to re-introduce code path involving Opstrace DNS service for internal dev cluster deployment.